frontend/src/metabase/querying/components/FilterValuePicker/FilterValuePicker.unit.spec.tsx

```diff
@@ -57,7 +57,7 @@ async function setupStringPicker({
     setupFieldSearchValuesEndpoints(result.field_id, value, result.values);
   });
 
-  renderWithProviders(
+  const { rerender } = renderWithProviders(
```

Comment:
Should this case be handled too?

https://github.com/metabase/metabase/assets/6830683/5aee86fa-e1a1-4baa-b8f2-ddb7445999a2

Reply:
Fixed, great catch!